lib/traject/annotation_config.rb

```diff
@@ -9,6 +9,7 @@
 end
 
 each_record do |record, context|
+  context.skip!("Skipping #{record} because it is not present") unless record
```

Comment:
This is just an extra-duper-safe guard clause?

Reply:
So.. maybe this is because of the issue of `each` vs. `map`, but what I found is that `True` objects were being passed through to traject and killing of the indexing.